enterprise/security/src/main/java/org/neo4j/server/security/enterprise/auth/AuthProcedures.java

```diff
@@ -427,6 +426,12 @@ private BoltConnectionTracker getBoltConnectionTracker()
         return graph.getDependencyResolver().resolveDependency( BoltConnectionTracker.class );
     }
 
+    private KernelTransaction getCurrentTx()
+    {
+        return graph.getDependencyResolver().resolveDependency( ThreadToStatementContextBridge.class )
+                .getKernelTransactionBoundToThisThread( true );
+    }
```

Comment:
Duplicated code from above. Perhaps there's a good class to add this as a `static` helper?